atd-vzd/README.md

```diff
@@ -29,16 +28,9 @@ hasura metadata apply --envfile .env.local
 - Start the local Hasura console and make any changes needed which will then reflect in your project folder
 ### Merging an approved feature branch
 
-We need to check the status of new migrations against the staging Hasura engine **before merging a feature branch** so that we can make updates to the migration version order if needed. The version refers to the timestamp in migration folder name.
-
-To check the status from your local project folder, run:
-```bash
-hasura migrate status --envfile .env.staging
-```
-
-Check that your new migration shows `NOT PRESENT` in the `DATABASE` column and that it is also the last row in the table shown. We need to make sure that migrations are applied in the same order.
+We need to check the order of migrations against those in the `master` **before merging a feature branch** so that we can make updates to the migration version order if needed. The version refers to the timestamp in migration folder name.
 
-If the new migration is not the last migration to apply in the sequence:
+To check migrations for any conflicts with latest migrations in the `master`` branch:
 - Make sure that your branch is up to date with `master`
 - Check to make sure no one else is actively merging their work and coordinate if needed
 - Update the migration version in your project so it is the newest migration
```

Comment:
the line after this `- Test locally using the steps in the  [Generating migrations and metadata changes section](#generating-migrations-and-metadata-changes)` confuses me bc is this rly how ur supposed to test that there are no conflicts? to me this just seems to be instructions on how to set up your env before creating any migrations

Reply:
Thanks for looking at this! This was really hard for me to capture in words, and I think that I was mostly trying to recycle a section instead of writing out more stuff. I pushed an edit, but I'm happy to continue making changes. I don't want this to be confusing. 🙏

Reply:
Yeah its hard to capture the migration process in words for sure :face_with_spiral_eyes:  I see now how recycling that section without writing out more stuff could make total sense for some ppl but I think the edit you just pushed makes it crystal clear! Esp if we ever get new ppl on the team, im always trying to think of newbies being able to understand the docs and im like well if im getting confused they might too :stuck_out_tongue_closed_eyes: